embulk-input-mysql/src/test/java/org/embulk/input/mysql/BasicTest.java

```diff
@@ -41,6 +42,7 @@ private static String readResource(String fileName)
     @Before
     public void setup()
     {
+        TimeZone.setDefault(TimeZone.getTimeZone("Europe/Helsinki")); // +200
```

Comment:
In my environment, `jvmArgs "-Duser.country=FI -Duser.timezone=Europe/Helsinki"` doesn't work.
Instead `jvmArgs "-Duser.country=FI", "-Duser.timezone=Europe/Helsinki"` works.
Is it because my gradle (2.12) is old?